.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/bendavid/narf.git
 [submodule "wremnants-data"]
 	path = wremnants-data
-	url = https://gitlab.cern.ch/cms-wmass/wremnants-data.git
+	url = https://gitlab.cern.ch/cms-wmass/wremnants-data.git
```

Comment:
In principle this file shouldn't be touched by this PR anymore at all (I think this is just a trivial change of whitespace)